aiapy/psf/psf.py

```diff
@@ -174,7 +174,7 @@ def filter_mesh_parameters(*, use_preflightcore=False):
 
 @u.quantity_input
 @validate_channel("channel", valid_channels=[94, 131, 171, 193, 211, 304, 335] * u.angstrom)
-def psf(channel: u.angstrom, *, use_preflightcore=False, diffraction_orders=None, use_gpu=True):
+def calculate_psf(channel: u.angstrom, *, use_preflightcore=False, diffraction_orders=None, use_gpu=True):
```

Comment:
Should we do a 1 release deprecation period?

Reply:
I added back psf and made it emit a warning.